oidc_cli/rsa-keygen.go

```diff
@@ -0,0 +1,55 @@
+package oidc
+
+import (
+	"fmt"
+	"crypto/rsa"
+	"crypto/rand"
+	"crypto/x509"
+	"golang.org/x/crypto/ssh"
+	"encoding/pem"
+	"gopkg.in/square/go-jose.v2"
+	"os"
+)
+
+func generateRSAKey() error {
```

Comment:
```suggestion
func generateRSAKey() (<>, error) {
```
We should return the result of jose.JWONWebKey here so that the caller can operate on the result.

Instead of simply `fmt.Println(string(b))`